python/ray/data/tests/test_stats.py

```diff
@@ -1136,6 +1144,24 @@ def test_write_ds_stats(ray_start_regular_shared, tmp_path):
     assert stats == ds._write_ds.stats()
 
 
+def test_time_backpressure(ray_start_regular_shared, restore_data_context):
+    class TimedBackpressurePolicy(BackpressurePolicy):
+        def __init__(self, topology: "Topology"):
+            self.start_time = time.perf_counter()
+
+        def can_add_input(self, op: "PhysicalOperator") -> bool:
+            return time.perf_counter() - self.start_time < 1
```

Comment:
I kinda thought I should have this sign flipped here (i.e. `time.perf_counter() - self.start_time > 1`), which to me would mean after 1 second we could add inputs. Before that it would block them creating backpressure. Can someone explain why the sign should be this way?

Reply:
yeah the way you describe it makes sense, and should be the intended approach. maybe the start_time is being initialized multiple times, and we need to make sure we're only setting it once?

another alternative could be to just have start time defined outside the class once, and reference it within can_add_input()

Reply:
Ok just wanted to make sure that I wasn't misunderstanding, I will play around with initializing the start time differently.